README.md

```diff
@@ -87,6 +87,8 @@ We are providing example apps for Swift and Objective-C. These apps demonstrate
 
 Gini Vision Library can either be installed by using CocoaPods or by manually dragging the required files to your project.
 
+**Note on iOS 10:** No matter which option you choose if you want to support iOS 10 you need to specify the `NSPhotoLibraryUsageDescription` key in your `Info.plist` file. And if you're using the [Gini iOS SDK](https://github.com/gini/gini-sdk-ios) you need to add support for "Keychain Sharing" in your entitlements by adding a `keychain-access-groups` value to your entitlements file.
```

Comment:
Maybe we can just point to the integration guide from the Gini SDK for more information.